[DOCS] Fix relative links on github

andyjessen opened the pull request:
This commit fixes relative links returning '404' error when accessed on https://github.com/great-expectations/great_expectations.

### Definition of Done
Please delete options that are not relevant.

- [X] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [X] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code

andyjessen committed:
Fix relative links on github

Fix relative links on github

This commit fixes relative links returning '404' error when accessed on https://github.com/great-expectations/great_expectations.

austiezr commented on docs/intro.md:
```diff
@@ -101,7 +101,7 @@ Slack is good for that, too: [join slack](https://greatexpectations.io/slack).
 
 #### Contribute code or documentation
-We welcome contributions to Great Expectations. Please start with our [Contributing guide](/docs/contributing/contributing) and don’t be shy with questions!
+We welcome contributions to Great Expectations. Please start with our [Contributing guide](../contributing.md) and don’t be shy with questions!
```
This should be `./contributing/contributing.md`

andyjessen replied:
I made the fix in new commit.

austiezr commented on docs/tutorials/getting_started/customize_your_deployment.md:
```diff
@@ -2,7 +2,7 @@
 ---
 
-At this point, you have your first, working local deployment of Great Expectations. You’ve also been introduced to the foundational concepts in the library: [Data Contexts](/docs/reference/data_context), [Datasources](/docs/reference/datasources), [Expectations](/docs/reference/expectations/expectations), [Profilers](/docs/reference/profilers), [Data Docs](/docs/reference/data_docs), [Validation] ...
+At this point, you have your first, working local deployment of Great Expectations. You’ve also been introduced to the foundational concepts in the library: [Data Contexts](../../reference/data_context.md), [Datasources](../../reference/datasources.md), [Expectations](../../reference/expectations/expectations.md), [Profilers](../../reference/profilers.md), [Data Docs](../../reference/data_docs.md ...
```
Validation link should be `../../reference/validation.md`

andyjessen replied:
I fixed the link.

austiezr commented on docs/tutorials/getting_started/customize_your_deployment.md:
```diff
@@ -15,88 +15,88 @@ This last section of this tutorial is designed to present you with clear options
-* [How to configure an Expectation store on a filesystem](/docs/guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_to_postgresql)
+* [How to configure an Expectation store in Amazon S3](../../guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_amazon_s3.md)
+* [How to configure an Expectation store in GCS](../../guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.md)
+* [How to configure an Expectation store in Azure Blob Storage](../..//guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_azure_blob_storage.md)
```
Extra `/`

andyjessen replied:
I fixed in another commit.

austiezr commented on docs/contributing/contributing_checklist.md:
```diff
@@ -48,9 +48,9 @@ Once your code is ready, please go through the following checklist before submit
+* See [Writing unit and integration tests](../contributing/contributing_test.md) for details on how to write unit tests in Great Expectations.
 
-* Please make certain to run `pytest` to verify that all tests pass locally. See [Running tests](/docs/contributing/contributing_test) for details.
+* Please make certain to run `pytest` to verify that all tests pass locally. See [Running tests](../contributing/contributing_test.md) for details.
```
`./contributing_test.md`

andyjessen replied:
I made the fix in new commit.

austiezr commented on docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md:
```diff
@@ -324,8 +324,8 @@ We will now take the local GE configuration from [Part 1](#part-1-local-configur
-1. [Running a Checkpoint in Airflow using a `bash operator`](/docs/deployment_patterns/how_to_use_great_expectations_with_airflow#option-1-running-a-checkpoint-with-a-bashoperator)
-2. [Running a Checkpoint in Airflow using a `python operator`](/docs/deployment_patterns/how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator)
+1. [Running a Checkpoint in Airflow using a `bash operator`](../deployment_patterns/how_to_use_great_expectations_with_airflow#option-1-running-a-checkpoint-with-a-bashoperator.md)
+2. [Running a Checkpoint in Airflow using a `python operator`](../deployment_patterns/how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator.md)
```
`./how_to_use_great_expectations_with_airflow#option-1-running-a-checkpoint-with-a-bashoperator.md`

`./how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator.md`

andyjessen replied:
I made the fix in new commit.

austiezr commented on docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md:
```diff
@@ -451,14 +451,14 @@ You've successfully migrated your Great Expectations configuration to Cloud Comp
 There are many ways to iterate and improve this initial version, which used a `bash operator` for simplicity. For information on more sophisticated ways of triggering Checkpoints, building our DAGs, and dividing our Data Assets into Batches using DataConnectors, please refer to the following documentation: 
 
-- [How to run a Checkpoint in Airflow using a `python operator`](/docs/deployment_patterns/how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator).
+- [How to run a Checkpoint in Airflow using a `python operator`](../deployment_patterns/how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator.md).
```
`./how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator.md`

andyjessen replied:
I made the fix in new commit.

austiezr commented on docs/guides/connecting_to_your_data/components/next_steps.md:
```diff
@@ -1,5 +1,5 @@
-- [How to create a Batch of data from an in-memory Spark or Pandas dataframe](/docs/guides/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_or_pandas_dataframe)
-- [How to create and edit Expectations with instant feedback from a sample Batch of data](/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data)
-- [How to create and edit Expectations based on domain knowledge, without inspecting data directly](/docs/guides/expectations/how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly)
+- [How to create a Batch of data from an in-memory Spark or Pandas dataframe](../../connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_or_pandas_dataframe.md)
```
`../connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_or_pandas_dataframe.md`

andyjessen replied:
I couldn't get suggested path to work.

austiezr commented on docs/guides/miscellaneous/how_to_write_a_how_to_guide.md:
```diff
@@ -128,24 +128,24 @@ Once a draft of your guide is written, you can see it rendered on a `localhost`
 4. Decide whether you’re writing a code-heavy or process-heavy guide, and adjust your formatting appropriately.
 
-5. Fill out the Prerequisites info box (see [How-to guide template file](/docs/guides/miscellaneous/how_to_template)). The header of the info box says: “This how-to guide assumes you have already:”. Place each prerequisite under its own bullet and phrase it using the style in the template: “done something” (e.g. "Configured a Datasource").
+5. Fill out the Prerequisites info box (see [How-to guide template file](../../guides/miscellaneous/how_to_template.md)). The header of the info box says: “This how-to guide assumes you have already:”. Place each prerequisite under its own bullet and phrase it using the style in the template: “done something” (e.g. "Configured a Datasource").
```
`./how_to_template.md`

andyjessen replied:
I made the fix in new commit.

austiezr commented on docs/guides/miscellaneous/migration_guide.md:
```diff
@@ -201,7 +201,7 @@ The V3-style Datasource has:
 </details>
 
-Migrating Datasource configurations that contain connections to the cloud or databases involve additional parameters like credentials that are specific to each configuration. The how-to-guides for Great Expectations contain numerous examples of V3 configurations that can be used for these various situations. Please check out our documentation on [Connecting to your Data](/docs/guides/connecting_t ...
+Migrating Datasource configurations that contain connections to the cloud or databases involve additional parameters like credentials that are specific to each configuration. The how-to-guides for Great Expectations contain numerous examples of V3 configurations that can be used for these various situations. Please check out our documentation on [Connecting to your Data](../../guides/connecting_t ...
```
`../connecting_to_your_data/index.md`

andyjessen replied:
I made the fix in another commit.

austiezr commented on docs/guides/miscellaneous/migration_guide.md:
```diff
@@ -232,7 +232,7 @@ The V3-style Datasource has:
 </details>
 
-Migrating Datasource configurations that contain connections to the cloud or databases involve additional parameters like credentials that are specific to each configuration. The how-to-guides for Great Expectations contain numerous examples of V3 configurations that can be used for these various situations. Please check out our documentation on [Connecting to your Data](/docs/guides/connecting_t ...
+Migrating Datasource configurations that contain connections to the cloud or databases involve additional parameters like credentials that are specific to each configuration. The how-to-guides for Great Expectations contain numerous examples of V3 configurations that can be used for these various situations. Please check out our documentation on [Connecting to your Data](../../guides/connecting_t ...
```
`../connecting_to_your_data/index.md`

andyjessen replied:
I made the fix in another commit.

austiezr commented on docs/guides/miscellaneous/migration_guide.md:
```diff
@@ -250,7 +250,7 @@ The V3-style Datasource has:
 :::note Note on Datasource in V3
 
-One exception to the datatype-agnostic Datasource in the V3 API is the <code>SimpleSqlalchemyDatasource</code>, which combines functionality of the <code>Datasource</code> and <code>ExecutionEngine</code> to enable [database introspection and partitioning](/docs/guides/connecting_to_your_data/how_to_configure_a_dataconnector_to_introspect_and_partition_tables_in_sql). More examples on using the < ...
+One exception to the datatype-agnostic Datasource in the V3 API is the <code>SimpleSqlalchemyDatasource</code>, which combines functionality of the <code>Datasource</code> and <code>ExecutionEngine</code> to enable [database introspection and partitioning](../../guides/connecting_to_your_data/how_to_configure_a_dataconnector_to_introspect_and_partition_tables_in_sql.md). More examples on using th ...
```
`../connecting_to_your_data/<path>`

andyjessen replied:
I made the fix in another commit.

austiezr commented on docs/guides/miscellaneous/migration_guide.md:
```diff
@@ -273,12 +273,12 @@ Migrating Datasource configurations that contain connections to databases involv
+- [How to connect to a MySQL database](../../guides/connecting_to_your_data/database/mysql.md)
+- [How to connect to a Redshift database](../../guides/connecting_to_your_data/database/redshift.md)
+- [How to connect to a Snowflake database](../../guides/connecting_to_your_data/database/snowflake.md)
+- [How to connect to a SQLite database](../../guides/connecting_to_your_data/database/sqlite.md)
```
`../connecting_to_your_data/<path>`

andyjessen replied:
I made the fix in another commit.

austiezr commented on docs/guides/miscellaneous/migration_guide.md:
```diff
@@ -291,7 +291,7 @@ Before doing the migration, we recommend that you create a backup of your V2 Che
 :::
 
-In Great Expectations version 0.13.7, we introduced an improved Checkpoints feature, which allowed Checkpoints to utilize features the V3 API. As a result, Checkpoints are now able to [filter and sort batches from configured datasources](/docs/guides/connecting_to_your_data/how_to_get_a_batch_of_data_from_a_configured_datasource), [introspect and partition tables as batches](/docs/guides/connecti ...
+In Great Expectations version 0.13.7, we introduced an improved Checkpoints feature, which allowed Checkpoints to utilize features the V3 API. As a result, Checkpoints are now able to [filter and sort batches from configured datasources](../..//guides/connecting_to_your_data/how_to_get_a_batch_of_data_from_a_configured_datasource.md), [introspect and partition tables as batches](../../guides/conn ...
```
`../connecting_to_your_data/<path>`

andyjessen replied:
I made the fix in another commit.

austiezr commented on docs/guides/miscellaneous/migration_guide.md:
```diff
@@ -349,8 +349,8 @@ Here is the equivalent configuration in V3-style. Notice that the Validation Ope
 
-- [How to add validations data or suites to a Checkpoint](/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint)
-- [How to configure a new Checkpoint using test_yaml_config](/docs/guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config)
+- [How to add validations data or suites to a Checkpoint](../../guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md)
```
`../validation/<path>`

andyjessen replied:
I made the fix in another commit.

austiezr commented on docs/guides/miscellaneous/migration_guide.md:
```diff
@@ -429,8 +429,8 @@ Here is the equivalent configuration in V3-style. Notice that the Validation Ope
-- [How to add validations data or suites to a Checkpoint](/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint)
-- [How to configure a new Checkpoint using test_yaml_config](/docs/guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config)
+- [How to add validations data or suites to a Checkpoint](../../guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md)
+- [How to configure a new Checkpoint using test_yaml_config](../../guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.md)
```
`../validation/<path>`

andyjessen replied:
I made the fix in another commit.

austiezr commented on docs/guides/miscellaneous/migration_guide.md:
```diff
@@ -508,8 +508,8 @@ Here is the equivalent configuration in V3-style. Notice that the Validation Ope
-- [How to add validations data or suites to a Checkpoint](/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint)
-- [How to configure a new Checkpoint using test_yaml_config](/docs/guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config)
+- [How to add validations data or suites to a Checkpoint](../../guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md)
+- [How to configure a new Checkpoint using test_yaml_config](../../guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.md)
```
`../validation/<path>`

andyjessen replied:
I made the fix in another commit.

austiezr commented on docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint.md:
`````diff
@@ -52,7 +52,7 @@ validations:
 ````
 
-This is the minimum required to configure a Checkpoint that will run the Expectation Suite `my_suite` against the data asset `MyDataAsset`. See [How to configure a new Checkpoint using test_yaml_config](/docs/guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config) for advanced configuration options.
+This is the minimum required to configure a Checkpoint that will run the Expectation Suite `my_suite` against the data asset `MyDataAsset`. See [How to configure a new Checkpoint using test_yaml_config](../../../guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.md) for advanced configuration options.
`````
`../../guides/<path>`

andyjessen replied:
I couldn't get suggested path to work.

austiezr commented on docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint.md:
```diff
@@ -97,9 +97,9 @@ Before running a Checkpoint, make sure that all classes and Expectation Suites r
-* [How to configure a new Checkpoint using test_yaml_config](/docs/guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config)
-* [How to add validations data or suites to a Checkpoint](/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint)
+* [How to configure a new Checkpoint using test_yaml_config](../../../guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.md)
+* [How to add validations data or suites to a Checkpoint](../../../guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md)
```
`../../guides/<path>`

andyjessen replied:
I couldn't get suggested path to work.

austiezr replied:
My bad; looks like these should be `./<file_name>`

austiezr commented on docs/guides/validation/validation_actions/how_to_collect_openlineage_metadata_using_a_validation_action.md:
```diff
@@ -16,7 +16,7 @@ This guide will explain how to use a Validation Action to emit results to an Ope
 
  - Created at least one Expectation Suite.
- - Created at least one [Checkpoint](/docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint) - you will need it in order to test that the OpenLineage Validation Operator is working.
+ - Created at least one [Checkpoint](../../../guides/validation/checkpoints/how_to_create_a_new_checkpoint.md) - you will need it in order to test that the OpenLineage Validation Operator is working.
```
`../../validation/<path>`

andyjessen replied:
I made the fix in another commit.

austiezr commented on docs/reference/data_discovery.md:
```diff
@@ -4,10 +4,10 @@ title: Data Discovery
 **observed** properties of data, Profiling helps to understand and reason about the data's **expected** properties.
 
-To produce a useful data overview, Great Expectations uses a [profiler](/docs/reference/profilers) to build a special Expectation
+To produce a useful data overview, Great Expectations uses a [profiler](../reference/profilers.md) to build a special Expectation
```
`./data_docs.md`

`./profilers.md`

andyjessen replied:
I made the fix in new commit.

austiezr commented on docs/reference/data_discovery.md:
```diff
@@ -28,4 +28,4 @@ Profiling is still a beta feature in Great Expectations. Over time, we plan to e
 consuming. As a rule of thumb, we recommend starting with small batches of data.
 
-See the docs on [Profilers](/docs/reference/profilers) for more information.
+See the docs on [Profilers](../reference/profilers.md) for more information.
```
`./profilers.md`

andyjessen replied:
I made the fix in new commit.

austiezr commented on docs/reference/validation.md:
```diff
@@ -6,7 +6,7 @@ title: Validation
 that details any specific deviations from expected values.
 
-We recommend using a [Data Context](/docs/reference/data_context) to manage Expectation Suites and coordinate validation across runs.
+We recommend using a [Data Context](../reference/data_context.md) to manage Expectation Suites and coordinate validation across runs.
```
`./data_context.md`

andyjessen replied:
I made the fix in new commit.

austiezr reviewed (changes requested):
Awesome to see this come in; thank you so much @andyjessen !

I've noted out places where the updated links are broken; otherwise love to see this. 🚀

andyjessen committed:
Modify relative link

andyjessen committed:
Fix relative links

andyjessen committed:
Fix relative links

austiezr committed:
Merge branch 'develop' into fix-github-links

austiezr committed:
Merge branch 'develop' into fix-github-links

austiezr committed:
Merge branch 'develop' into fix-github-links

austiezr commented on docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md:
```diff
@@ -324,8 +324,8 @@ We will now take the local GE configuration from [Part 1](#part-1-local-configur
-1. [Running a Checkpoint in Airflow using a `bash operator`](/docs/deployment_patterns/how_to_use_great_expectations_with_airflow#option-1-running-a-checkpoint-with-a-bashoperator)
-2. [Running a Checkpoint in Airflow using a `python operator`](/docs/deployment_patterns/how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator)
+1. [Running a Checkpoint in Airflow using a `bash operator`](./how_to_use_great_expectations_with_airflow#option-1-running-a-checkpoint-with-a-bashoperator.md)
+2. [Running a Checkpoint in Airflow using a `python operator`](./how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator.md)
```
`./<path>.md#<heading>`

andyjessen replied:
I fixed link in new commit.

austiezr commented on docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md:
```diff
@@ -451,14 +451,14 @@ You've successfully migrated your Great Expectations configuration to Cloud Comp
 There are many ways to iterate and improve this initial version, which used a `bash operator` for simplicity. For information on more sophisticated ways of triggering Checkpoints, building our DAGs, and dividing our Data Assets into Batches using DataConnectors, please refer to the following documentation: 
 
-- [How to run a Checkpoint in Airflow using a `python operator`](/docs/deployment_patterns/how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator).
+- [How to run a Checkpoint in Airflow using a `python operator`](./how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator.md).
```
`./<path>.md#<heading>`

andyjessen replied:
I fixed link in new commit.

austiezr replied:
Hey @andyjessen ! I'm not seeing the latest commit re: above.

andyjessen replied:
Thanks @austiezr for catching my mistake. (I made commit on separate branch that got pruned.) I added to current branch and pushed.

austiezr reviewed (changes requested):
I believe these are the last pieces; thank you for your effort on this! 🙇

andyjessen commented:
> I believe these are the last pieces; thank you for your effort on this! 🙇

Hi @austiezr  - I fixed links. I noticed another broken path in same file and fixed it as well.

austiezr committed:
Merge branch 'develop' into fix-github-links

andyjessen committed:
fix links

austiezr reviewed (approved):
Killer, thank you! 🙇

austiezr committed:
Merge branch 'develop' into fix-github-links
